### PR TITLE
[#741] Fix opera CSS

### PR DESCRIPTION
**Pull request by stanica**
Fixes the CSS in opera so that the page looks the same as in Chrome.

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/ed2d295549f5983/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/ed2d295549f5983/output.txt

Total script time: 0.27 mins

File: css/butter.ui.deprecated.css [hunk 1]
```diff
@@ -188,23 +253,27 @@ input.form-error {
...
-  left: 0px;
-  top: 0px;
+  left: 0;
+  top: 0;
   height: 35px;
   color: #fff;
-  background-image: -moz-linear-gradient( center bottom, #2C2C2C 0, #4C4C4C 100% );
   background-image: -webkit-gradient( linear, left bottom, left top, color-stop( 0, #2C2C2C ), color-stop( 1, #4C4C4C ) );
+  background-image: -o-linear-gradient( top, #4C4C4C 0%, #2C2C2C 100% );
+  /* XXXrobert - should audit MS's use of center bottom */
```
Comment (jbuck) [inline css/butter.ui.deprecated.css:250]: File a ticket on lighthouse for this if you haven't already

**Comment by jbuck** [comment]
One style nit, add two more command-line parameters to the csslint invocation:

``` diff
diff --git a/make.js b/make.js
index 0977aea..444c978 100755
--- a/make.js
+++ b/make.js
@@ -96,6 +96,7 @@ target['check-css'] = function() {

   exec(CSSLINT + ' --warnings=' + warnings +
                  ' --errors=' + errors +
+                 ' --quiet --format=compact' +
                  ' ' + CSS_DIR +
                  ' ' + DIALOGS_DIR +
                  ' ' + TEMPLATES_DIR);

```

**Comment by jbuck** [comment]
Doing that will make /botio check pass

File: css/butter.ui.deprecated.css [hunk 2]
```diff
@@ -17,15 +31,50 @@
...
+    -ms-transform:rotate(0deg);
+    transform:rotate(0deg);
+  }
+  100% {
+    -ms-transform:rotate(360deg);
+    transform:rotate(360deg);
+  }
+}
+
+@-o-keyframes spin {
```
Comment (jbuck) [inline css/butter.ui.deprecated.css]: lets yank this out until Opera 12 is officially out and csslint supports its new features

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @dseif received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/26df9a27d354bbc/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/26df9a27d354bbc/output.txt

Total script time: 0.39 mins

File: make.js [hunk 3]
```diff
@@ -54,6 +58,48 @@ target.docs = function() {
...
+    "errors",
+    "gradients",
+    "font-faces",
+    "floats",
+    "vendor-prefix"
+  ].join(",");
+
+  exec(CSSLINT + ' --warnings=' + warnings +
+                 ' --errors=' + errors +
+ 		 ' --quiet --format=compact' +
```
Comment (jbuck) [inline make.js]: Fix the alignment

**Pull request merged**